Cargo.toml

```diff
@@ -34,12 +34,15 @@ categories = ["command-line-utilities", "development-tools"]
 
 [dependencies]
 aho-corasick = "1.0.2"
+ansi-to-tui = "3.1.0"
```

Comment:
Converts a byte slice into `ratatui`'s `Text` type, which represents formatted text. Without this, the escape codes are printed to the screen as regular characters.